### PR TITLE
[OV-1695] Add transactions logs to certificate model

### PR DESCRIPTION
**Pull request by JonaszJestem**

### CODE REVIEW

**Review by soanvig — CHANGES_REQUESTED** [review]
@JonaszJestem  please add documentation in readme

File: packages/origin-247-certificate/test/offchain-certificate/offchain-certificate.spec.ts [hunk 1]
```diff
@@ -407,6 +409,40 @@ const getFailingModuleForUnitTests = (failingOn: 'issue' | 'claim' | 'transfer',
...
+        public async batchClaimWithTxHash(command: IClaimCommand[]) {
+            await this.batchClaim();
+
+            return { transactionHash: 'txHash' };
+        }
+
+        public async issueWithTxHash(command: IIssueCommandParams<T>) {
+            await this.issue();
+
+            return { certificate: {} as any, transactionHash: 'txHash' };
```
Comment (soanvig) [inline packages/origin-247-certificate/test/offchain-certificate/offchain-certificate.spec.ts:432]: pleeeease return correct certificate at least lol

**Review by soanvig — CHANGES_REQUESTED** [review]

File: packages/origin-247-certificate/test/offchain-certificate/offchain-certificate.spec.ts (hunk 1 above)
Comment (JonaszJestem) [reply packages/origin-247-certificate/test/offchain-certificate/offchain-certificate.spec.ts]: fixed

File: packages/origin-247-certificate/test/offchain-certificate/offchain-certificate.spec.ts [hunk 2]
```diff
@@ -427,9 +428,34 @@ const getFailingModuleForUnitTests = (failingOn: 'issue' | 'claim' | 'transfer',
         }
 
         public async issueWithTxHash(command: IIssueCommandParams<T>) {
-            await this.issue();
+            const { id } = await this.issue();
 
-            return { certificate: {} as any, transactionHash: 'txHash' };
+            return {
```
Comment (soanvig) [inline packages/origin-247-certificate/test/offchain-certificate/offchain-certificate.spec.ts:433]: can't you just take certificate returned from `issue`? I don't understand

**Review by soanvig — CHANGES_REQUESTED** [review]

**Review by soanvig — APPROVED** [review]

**Pull request merged**